test/asynchronous/test_client.py

```diff
@@ -31,13 +32,15 @@
 import sys
 import threading
 import time
-from typing import Iterable, Type, no_type_check
+import uuid
+from typing import Any, Iterable, Type, no_type_check
 from unittest import mock
 from unittest.mock import patch
 
 import pytest
 import pytest_asyncio
 
+from bson.binary import *
```

Comment:
We try to avoid * imports for a few reasons: https://stackoverflow.com/questions/2386714/why-is-import-bad